deposit_contract/tests/contracts/utils.py

```diff
@@ -0,0 +1,16 @@
+import json
+import os
+
+DIR = os.path.dirname(__file__)
+
+
+def get_deposit_contract_code():
+    file_path = os.path.join(DIR, './../../contracts/validator_registration.v.py')
+    deposit_contract_code = open(file_path).read()
+    return deposit_contract_code
+
+
+def get_deposit_contract_json():
+    file_path = os.path.join(DIR, './../../contracts/validator_registration.json')
```

Comment:
If we remove the ABI from the repo, we can build this dynamically at the start of testing instead

Reply:
I see the benefits of having ABI in the repo are:
1. Convenience: the client devs can just use the ABIs from the repo without compiling.
2. Versioning: to make sure we are using the consistent Vyper version.